boot/linux-x86_64/BUILD.go

```diff
@@ -5,12 +5,19 @@ import (
 	"dbt-rules/RULES/core"
 	"dbt-rules/RULES/util"
 
-	"libsupcxx/RULES/platform"
+	"libsupcxx"
 	"libsupcxx/libsupcxx/include"
 
 	gcc "x86_64-elf-gcc"
 )
 
+var bootToolchain = gcc.Toolchain.NewWithStdLib(
```

Comment:
This probably needs an explanation. I want the boot libs to be compiled with the same toolchains as the other libs (before they used the one straight gcc package, which is pretty weird). The problem is that if I try to say `Toolchain: Toolchain` to make sure that only the right toolchain is used, go complains that the `Toolchan` struct and the `boot*` libs are circularly dependent. This bootToolchain is used to fix that, breaking the circular dependency.

Reply:
> The whole point is that they should not use standard headers!

I meant the std headers that come from libsupcxx, i.e. the same ones as the other libs.

Reply:
(I can see where the confusion came from, I will change the commit comment for future reference before merging)